test/controllers/IndividualAddressControllerSpec.scala

```diff
@@ -218,5 +219,35 @@ class IndividualAddressControllerSpec extends SpecBase with MockitoSugar with Nu
 
       application.stop()
     }
+
+    "must redirect to the Check your answers page when user change their answer" in {
+
+      val mockSessionRepository = mock[SessionRepository]
+      val individualDateOfBirthRouteCheck = routes.IndividualAddressController.onPageLoad(CheckMode)
+      val userAnswers = UserAnswers(userAnswersId).set(IndividualAddressPage, validAnswer).success.value
+
+      when(mockSessionRepository.set(any())) thenReturn Future.successful(true)
+
+      val application =
+        applicationBuilder(userAnswers = Some(userAnswers))
+          .overrides(
+            bind[Navigator].toInstance(new FakeNavigator(routes.IndividualCheckYourAnswersController.onPageLoad(), mode = CheckMode)),
+            bind[SessionRepository].toInstance(mockSessionRepository)
+          )
+          .build()
+
+      val request =
+        FakeRequest(POST, individualDateOfBirthRouteCheck.url)
+          .withFormUrlEncodedBody(validData.toList:_*)
+
```

Comment:
Maybe rename the unit test? Answer isn't changed but the mode is changed to CheckMode. You can also move this test to NavigatorSpec?

Reply:
Moved tests to CheckModeNavigatorSpec.